src/store/pet/models.ts

```diff
@@ -1,11 +1,10 @@
-import { PetProfile } from '../shared/models';
+import { PetProfile } from '../../shared/models';
```

Comment:
Wouldn't be better to rename `src/store/pet/models.ts` file to `src/store/pet/pet.models.ts`?

Reply:
it would:)